### DISCUSSION TITLE
Package section to contain latest version number

### DISCUSSION THREAD

**Discussion by tanguyantoine** [community/community#5785] [Actions]
# What

As a Github registry user that downloads packages, I would like to have the version number displayed in the `Packages` sidebar section of a repo so that I know quickly if new version are available

Something similar to this 
![image](https://user-images.githubusercontent.com/263097/134655238-543f5485-6f6a-4b71-8af4-f2715ab73fc9.png)

**Comment by kg-currenxie** [comment]
Totally agree, but since the sidebar only shows a few of them, I'd like to have it in the full list as well:


![Screenshot 2021-10-12 at 13 23 22](https://user-images.githubusercontent.com/48229166/136896579-44b1cbad-d2ea-4711-837c-bb04c069743f.png)

If I want to bulk-update all my (9) packages, I have to click on each package to check the latest version. Quite annoying :) Thanks.
